Competition/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -22,11 +23,12 @@
     public XCANTalon master;
     public XCANTalon follower;
     public XDigitalInput calibrationSensor;
-    private final DoubleProperty currentCalibrationSensorPosition;
+    public XSolenoid activateBrakeSolenoid;
```

Comment:
Nit: Consider making this `final` as well so we are always sure we're assigning to it in the constructor. Same thing for the other sensors/actuators above actually.

Reply:
if I make the other sensors/actuators the constructor thinks that they haven't been instantiated since they're wrapped around an if statement.

changed XSolenoid to final

Reply:
we can still do final, just in the else clause we can explicitly set them to null I think?